.gitignore

```diff
@@ -3,3 +3,6 @@
 
 # Local cache of Rubocop remote config
 .rubocop-*
+
+# Security stuff
+twillio_creds.sh
```

Comment:
I didn't get this far so might be correct but looks like a typo of twilio

Reply:
Yes, that is indeed a typo, will change that. Thanks!